CollectApiData.js

```diff
@@ -25,9 +25,11 @@ function datacollect() {
     };
 
     myConnector.getData = function (table, doneCallback) {
-		var data = JSON.parse(tableau.connectionData);
+		    var data = JSON.parse(tableau.connectionData);
 
-        var url_json = 'https://api.searchmetrics.com/v4/' + tableau.connectionName + '.json?' + data.params;
+		    const newParams = processExpressions(data.params);
```

Comment:
here we replace expressions with real data. It's executed on every data fetch.

Reply:
It worked well. Thank you Rey